marlowe-tutorial/doc/introducing-marlowe.adoc

```diff
@@ -0,0 +1,116 @@
+
+All the constructs of Marlowe that require user participation – including user deposits and user choices – are protected by timeouts. Because of this, it is easy to see that the *commitment* made by a participant to a contract is _finite_: we can predict when the contract will have nothing left to do: at this point any unspent funds are refunded to participants, and the contract stops, or _terminates_. So, any funds put into the contract by a participant _can't be locked up forever_: at this point the commitment effectively ends. 
+
+What is more, it is easy for us to _read off_ from the contract when it will terminate, and so all participants will know this before taking part in the contract,
+
+In our model, a running contract cannot force a deposit or a choice
+to happen: all it can do is to request a deposit or choice from a participant. In other words, for these actions it cannot
+“__push__”, but it can “__pull__”. On the other hand, it _can_ make payments automatically, so some aspects of a Marlowe contract can “push”to make some things happen, e.g. ensuring that a payment is made to a participant by constructing an appropriate transaction output.
+
+
+=== Marlowe in action
+
+We are working on a production release of Marlowe on the Cardano
+blockchain later in 2019 / early 2020. From today, you are able to explore
+Marlowe for yourself, either by downloading it and using the Haskell
+implementation directly, or by using the online Marlowe Playground
+simulation tool; these are both covered in subsequent tutorials. These
+will also cover the details of Marlowe, introduce a series of examples,
+look deeper into the tools for Marlowe.
+
+In the next six months we will be polishing the language design itself
+and developing a set of templates for popular financial instruments. Because Marlowe is a DSL we can work out how Marlowe contracts will behave without running them: this means that we can provide valuable diagnostics to potential participants before they commit to a contract, using _static analysis_. We can also use logic tools to _formally prove properties_ of Marlowe
+contracts, giving users the highest level of assurance that their
+contracts behave as intended.
+
+=== Where to go to find out more
+
+This tutorial covers the latest version of Marlowe, while all but the
+first of these links covers Marlowe 1.3, which is very close to Marlowe
+3.0 in conception but which differs in a number of detailed ways. 
+
+* The Marlowe github repository:
+https://github.com/input-output-hk/marlowe[link]
+* The Marlowe paper: https://iohk.io/research/papers/#2WHKDRA8[link]
```

Comment:
Better to link to the published version? Not sure how stable those library links are.